app/controllers/prometheus/create.js

```diff
@@ -276,4 +277,28 @@ export default class PrometheusCreateController extends PrometheusController {
      */
     afterCancel() {
     }
+
+    /**
+     * This function is used to call generateSchemaFromMeta() utility function in order to generate schema
+     * by passing controller's metadata and then pass that array of schemas to setSchemas() method in order 
+     * to set each schema in the respective class object.
+     * 
+     * @method setupSchema
+     * @protected
+     */
+    setupSchema() {
```

Comment:
Should this be done on the app level controller?

Reply:
If we want validations to be done other than creating/editing, then yes then should be inside the _AppController_. Otherwise, this should be inside _PrometheusCreateController_ where our validation function exists.